src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -49,6 +60,57 @@ internal bool AutomaticRedirection
                 _automaticRedirection = value;
             }
         }
+        internal bool UseProxy
+        {
+            get
+            {
+                if (_proxyPolicy == CurlProxyUsePolicy.UseDefaultProxy)
+                {
+                    return false;
```

Comment:
This is not correct. See comments below about proper semantics of .UseProxy property.